ansible/roles/kubernetes/tasks/debian.yml

```diff
@@ -39,65 +106,18 @@
     - 'kubelet'
     - 'kubeadm'
 
-- name: add apt signing key for helm (1/3)
-  apt_key:
-    url: https://baltocdn.com/helm/signing.asc
-    state: present
-
-- name: adding apt repository for helm (2/3)
-  apt_repository:
-    repo: deb https://baltocdn.com/helm/stable/debian/ all main
-    state: present
-  register: helm_repository
-  retries: 10
-  until: helm_repository is success
-
-- name: install helm package (3/3)
-  apt:
-    name: helm
-    state: present
-    force: true
-    update_cache: true
-  register: apt_install_helm
-  retries: 5
-  until: apt_install_helm is success
-
-# Adding required Kubernetes cgroups
+# TODO: Move to correct location; Perhaps common for RPI?
+# Adding required Kubernetes cgroups for Raspberry Pi
 - name: Enable container features
   ansible.builtin.replace:
     path: '{{ common_rpi_cmd_file }}'
     regexp: '^([\w](?!.*\b{{ item }}\b).*)$'
     replace: '\1 {{ item }}'
   with_items:
     - 'cgroup_enable=cpuset'
-    - 'cgroup_memory=1'
     - 'cgroup_enable=memory'
+    - 'cgroup_memory=1'
+    - 'swapaccount=1'
```

Comment:
Is `swapaccount=1` necessary? thought that was a Docker thing?

Reply:
Correct, although adding it in the case some people use that combination rather than creating it as a condition